src/Backend/Core/Js/Components/Resize.js

```diff
@@ -0,0 +1,29 @@
+import requestAnimationFrame from 'requestanimationframe'
+
+export class Resize {
+  constructor () {
+    this.ticking = false
+
+    this.calculate = () => {
+      window.backend.navigation.resize()
+
+      this.ticking = false
+    }
+
+    this.resize()
+  }
+
+  resize () {
+    $(window).on('load resize', () => {
+      console.log('RESIZE')
```

Comment:
👀 

also, does it need a debouncer? If I resize my browser then 200+ events are easily sent

Reply:
That is what the polyfill does, that I have imported at the top of the file, isn't it? https://www.paulirish.com/2011/requestanimationframe-for-smart-animating/

Reply:
Aha! Didn't look that up 😁 ✅